src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -151,51 +151,34 @@ Array [
 exports[`components/IRS Reports/IRSReportingMap renders correctly: Response item ProgressBar 1`] = `
 Array [
   <ProgressBar
+    animate={false}
+    cssClass="progress-bar-striped"
     decimalPoints={0}
     height="10px"
-    indicatorThresholds={
+    lineColor="#0000FF"
+    lineColorThresholds={
       Object {
-        "GREEN_THRESHOLD": Object {
-          "color": "#2ECC40",
-          "name": "Green",
-          "value": 1,
-        },
-        "GREY_THRESHOLD": Object {
-          "color": "#dddddd",
-          "name": "Grey",
-          "value": 0.2,
-        },
-        "RED_THRESHOLD": Object {
-          "color": "#FF4136",
-          "name": "Red",
-          "orEquals": true,
-          "value": 0.75,
-        },
-        "YELLOW_THRESHOLD": Object {
-          "color": "#FFDC00",
-          "name": "Yellow",
-          "value": 0.9,
-        },
+        "#2ECC40": 100,
+        "#FF4136": 75,
+        "#FFDC00": 90,
+        "#dddddd": 20,
       }
     }
     max={100}
     min={0}
+    showLabel={false}
+    stripped={false}
     value={50}
   >
     <div
       className="progress"
-      style={
-        Object {
-          "height": "10px",
-          "marginBottom": "15px",
-        }
-      }
```

Comment:
Does the removal of this imply that we no longer need this CSS?

Reply:
We do, height is also a prop in the js library  that had a default value of `10px`